packages/algob/src/lib/algo-operator.ts

```diff
@@ -278,7 +278,7 @@ export class AlgoOperatorImpl implements AlgoOperator {
       flags.localBytes,
       flags.globalInts,
       flags.globalBytes,
-      parseSSCAppArgs(flags.appArgs),
+      parseSSCAppArgs(flags.appArgs), // parseSSCAppArgs returns undefined if flags.appArgs is undefined
```

Comment:
we don't need this comment here. It was important to add it to the function doc string.

Reply:
Ok